### DISCUSSION TITLE
Welcome to COOS Discussions!

### DISCUSSION THREAD

**Discussion by FranckCo** [linked-statistics/COOS#20] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DrJMunozMx** [comment]
Hello everyone, nice to be in this community. A little introduction of myself:
Juan Muñoz; PhD on Computer Science; Director of IT Planning and Standardisation at INEGI, the National Institue of Statistics and Geography of Mexico
